prime-router/src/test/kotlin/cli/LookupTableCommandsTest.kt

```diff
@@ -86,116 +158,84 @@ class LookupTableCommandsTest {
-
-        // A response with an error
-        every { mockResultFailure.error.response.body().asString(HttpUtilities.jsonMediaType) } returns
-            """{"error": "some error"}"""
-        assertThat(LookupTableEndpointUtilities.getErrorFromResponse(mockResultFailure))
-            .isEqualTo("some error")
-    }
-
-    @Test
-    fun `check common errors from response test`() {
-        val mockResult = mockk<Result.Success<FuelJson>>()
-        val mockResultFailure = mockk<Result.Failure<FuelError>>()
-        val mockResponse = mockk<Response>()
-
-        // API Not found
-        every { mockResponse.statusCode } returns HttpStatus.SC_NOT_FOUND
-        every { mockResultFailure.error.response.body().isEmpty() } returns false
-        every { mockResultFailure.error.response.body().asString(HttpUtilities.jsonMediaType) } returns "not Json"
-        assertFailsWith<IOException>(
-            block = {
-                LookupTableEndpointUtilities.checkCommonErrorsFromResponse(mockResultFailure, mockResponse)
-            }
-        )
-
-        // API Not found with unexpected JSON response
-        every { mockResultFailure.error.response.body().asString(HttpUtilities.jsonMediaType) } returns
-            """{"dummy": "dummy"}"""
-        assertFailsWith<IOException>(
-            block = {
-                LookupTableEndpointUtilities.checkCommonErrorsFromResponse(mockResultFailure, mockResponse)
-            }
-        )
-
-        // Table not found
-        every { mockResultFailure.error.response.body().asString(HttpUtilities.jsonMediaType) } returns
-            """{"error": "some error"}"""
-        assertFailsWith<LookupTableEndpointUtilities.Companion.TableNotFoundException>(
+    fun `get table info from response create table conflict`() {
+        val exception = assertFailsWith<LookupTableEndpointUtilities.Companion.TableConflictException>(
+            message = "Expect TableConflictException does not thrown.",
             block = {
-                LookupTableEndpointUtilities.checkCommonErrorsFromResponse(mockResultFailure, mockResponse)
+                val respBody = """{"error": "New Lookup Table Table is identical to existing table version 1."}"""
+                getMockUtil(
+                    "${LookupTableEndpointUtilities.endpointRoot}/race",
+                    HttpStatusCode.Conflict,
+                    body = respBody
+                ).createTable("race", listOf(mapOf()), true)
             }
         )
-
-        // Nome other error
-        every { mockResponse.statusCode } returns HttpStatus.SC_BAD_REQUEST
-        assertFailsWith<IOException>(
-            block = {
-                LookupTableEndpointUtilities.checkCommonErrorsFromResponse(mockResultFailure, mockResponse)
-            }
+        assertTrue(
+            exception.message.toString().contains(
+                "New Lookup Table Table is identical to existing table version 1"
+            )
         )
+    }
```

Comment:
Its better to assert the exception type than the message within

Reply:
There's an assertion above on the exception type. This assertion is further verifying the message content which seems appropriate in the context. We should make that exception unique but feels out of scope for now.